dips/tests/test_migrations.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.models import Group
```

Comment:
Yes, it will definitely require changes if we modify the migrations but I think it's a good idea to test the backward methods in the existing ones. Adapting it to the migrations in the future should not be a big deal, we may squash the migrations before we do a final release, but I think that's not something we'll do often in the future.

Reply:
OK, I mean it doesn't hurt anything.